Makefile

```diff
@@ -153,7 +153,7 @@ crossdock-fresh: install
 	docker-compose rm --force
 	docker-compose pull
 	docker-compose build
-	docker-compose run crossdock
+	docker-compose up crossdock
```

Comment:
Why did you switch this? `docker-compose up` won't exit when the container does, whereas `docker-compose run` will.